test/integration/minishift.go

```diff
@@ -263,6 +263,15 @@ func (m *Minishift) checkServiceRolloutForSuccess(service string, timeout int, d
 		})
 	m.mutex.Unlock()
 
+	// get application's build logs
+	command = fmt.Sprintf("logs bc/%s", service)
```

Comment:
@budhrg This is going to happen irrespective of pass or fail, isn't it? Should we try to make it optional i.e. enable it somehow if you want to get more information?

Reply:
Done @LalatenduMohanty